docker-compose/test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# start docker daemon
+source /docker-lib.sh
+start_docker
+
+echo ${DOCKER_PASSWORD} | docker login -u ${DOCKER_USERNAME} --password-stdin
+unset DOCKER_USERNAME
+unset DOCKER_PASSWORD
+
+set -e
+ARGV_DIRECTORY="$1"
+set -u
+
+[[ "${DEBUG}" == "false" ]] || set -x
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+pushd $ARGV_DIRECTORY
+
+sut=$(yq read repo.yml 'sut')
```

Comment:
what does `sut` stand for?

Reply:
I think it stands for service under test, it comes from https://docs.docker.com/docker-hub/builds/automated-testing/ Think about it, it is not a great name, as its the service running the tests, rather than the one(s) under tests

Reply:
I agree with you, `sut` is not a great name, as it was confusing for at least two people (@roman-mazur and me). What about renaming it explicitly to `service_under_test`? :)

Reply:
For now, I would rather stick to the conventions used in Docker.

Reply:
From the link I sense that is an example (and unfortunate) service name rather than a convention... and I was wrong, I would call it `test_runner` instead of `service_under_test`

Reply:
It is a convention:
```
 You can define any number of linked services in this file. The only requirement is that sut is defined. Its return code determines if tests passed or not. Tests pass if the sut service returns 0, and fail otherwise.
```

Reply:
@roman-mazur @kaisoz I added the possibility to override the default from repo.yml, but if that is not specified we will assume sut

Reply:
Sounds good.

Reply:
Just got all the notifications at once... I don't know what happened...

@roman-mazur You are right, Docker requires that name. I don't know what I read that day, my bad...

@nazrhom That sounds awesome :+1: